packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -517,6 +515,38 @@ function ScatterplotViz(props: VisualizationProps<Options>) {
     (vizConfig.valueSpecConfig === 'Smoothed mean with raw' ||
       vizConfig.valueSpecConfig === 'Best fit line with raw');
 
+  // non-zero baseline for continuous overlay variable
+  const defaultOverlayMinMax = useDefaultAxisRange(
+    overlayVariable,
+    undefined,
+    undefined,
+    undefined,
+    vizConfig.dependentAxisLogScale,
```

Comment:
I'm not sure we should be passing `dependentAxisLogScale` for the "overlay scale" range?  `false` should be OK?

Reply:
I think you are probably right. Passing false would be good enough.